src/databricks/labs/blueprint/wheels.py

```diff
@@ -232,6 +232,26 @@ def upload_to_wsfs(self) -> str:
             self._installation.save(Version(self._product_info.version(), remote_wheel, self._now_iso()))
             return remote_wheel
 
+    def upload_wheel_dependencies(self, prefixes: list[str] | None = None) -> list[str]:
+        """Uploads the wheel dependencies to WSFS location of installation and returns the remote paths.
+        :param prefixes (optional): A list of prefixes to match against the wheel names. If a prefix matches, the wheel is uploaded. If None, all wheels are uploaded. Defaults to None.
+        """
+        if prefixes is None:
+            prefixes = [wheel.name for wheel in self._local_wheel]
+        remote_paths = []
+        for wheel in self._local_wheel:
```

Comment:
```suggestion
        for wheel in self._build_wheel(self._tmp_dir.name, verbose=self._verbose, no_deps=False):
```
this is what you need.